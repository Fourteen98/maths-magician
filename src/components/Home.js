function Home() {
  return (
    <div className="home">
      <h2>Welcome to our page!</h2>
      <p>
        You can have your maths calculations here.
      </p>
    </div>
  );
}
export default Home;
