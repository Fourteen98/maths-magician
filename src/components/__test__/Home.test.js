import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('renders when there is item', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
