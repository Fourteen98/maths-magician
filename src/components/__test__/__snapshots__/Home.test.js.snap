// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there is item 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    You can have your maths calculations here. Go to the Calculator section from the Navbar to get started.
  </p>
</div>
`;
